backend/core/src/listings/listings.service.ts

```diff
@@ -15,6 +15,7 @@ export class ListingsService {
       .leftJoinAndSelect("listings.leasingAgents", "leasingAgents")
       .leftJoinAndSelect("listings.preferences", "preferences")
       .leftJoinAndSelect("listings.property", "property")
+      .leftJoinAndSelect("listings.applications", "applications")
```

Comment:
With current TypeORM performance issues this will mostly likely cause container run OOM on heroku. Probably a workaround for that would be doing a separate query with `count`.